hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/Create2OperationSuite.java

```diff
@@ -625,9 +627,9 @@ private HapiSpec canMergeCreate2MultipleCreatesWithHollowAccount() {
         final AtomicReference<ByteString> partyAlias = new AtomicReference<>();
 
         return propertyPreservingHapiSpec("CanMergeCreate2MultipleCreatesWithHollowAccount")
```

Comment:
I guess `preserving`/`overriding` don't check that the property actually exist?  Because otherwise there'd be a merge-order dependency between this and #7425?